src/pages/organizers/[organizerId]/preview/OrganizerTable.tsx

```diff
@@ -0,0 +1,236 @@
+          flex={1}
+          css={`
+            white-space: pre-wrap;
+          `}
+        >
+          {images?.map((image) => (
+            <Inline
+              key={image['@id']}
+              spacing={4}
+              paddingTop={3}
+              paddingBottom={3}
+              css={`
+                &:not(:last-child) {
+                  border-bottom: 1px solid ${grey2};
+                }
+              `}
+            >
+              <Inline>
+                <Link href={image.contentUrl}>
+                  <Image
+                    width={300}
+                    src={image.thumbnailUrl}
+                    alt={image.description}
+                  />
+                </Link>
+              </Inline>
+              <Stack>
+                {isMainImage(image.thumbnailUrl) && (
+                  <Text
+                    backgroundColor={udbMainDarkGrey}
+                    color="white"
+                    alignSelf="flex-start"
+                    borderRadius="3px"
+                    paddingRight={3}
+                    paddingLeft={3}
+                    fontSize="0.8rem"
+                    fontWeight="bold"
+                  >
+                    {t('organizers.detail.mainImage')}
+                  </Text>
+                )}
+                <Text>{image.description}</Text>
+                <Text color={udbMainDarkGrey}>
+                  {`© ${image.copyrightHolder}`}
+                </Text>
+              </Stack>
+            </Inline>
+          ))}
+        </Stack>
+      </Inline>
+    );
+  };
+
+  const renderOrganizerLabels = (title: string, organizer: Organizer) => {
+    return (
+      <Inline
+        padding={3}
+        css={`
+          border-bottom: 1px solid ${grey2};
+        `}
+      >
+        <Text minWidth="15rem" color={udbMainDarkGrey}>
+          {t(title)}
+        </Text>
+        <OrganizerLabels organizer={organizer} />
+      </Inline>
+    );
+  };
+
+  return (
+    <Stack
+      flex={1}
+      padding={4}
+      marginBottom={5}
+      borderRadius="0.5rem"
+      backgroundColor="white"
+      css={`
+        box-shadow: ${getGlobalValue('boxShadow.medium')};
+      `}
+    >
+      {renderOrganizerInfo('organizers.detail.name', formattedName)}
+      {formattedDescription &&
+        renderOrganizerInfo(
+          'organizers.detail.description',
+          formattedDescription,
+        )}
+      {formattedEducationalDescription &&
+        renderOrganizerInfo(
+          'organizers.detail.educationalDescription',
+          formattedEducationalDescription,
+        )}
+      {renderOrganizerInfo('organizers.detail.address', formattedAddress)}
+      {renderOrganizerInfo(
+        'organizers.detail.contact',
+        formattedEmailAndPhone,
+        organizer?.contactPoint?.url,
+      )}
+      {renderOrganizerLabels('organizers.detail.labels', organizer)}
+      {renderOrganizerImages('organizers.detail.images', organizer?.images)}
```

Comment:
While this works in terms of end result, I believe it does mean it's harder for React to compute changes and re-render efficiently. So I would maybe make those small components, they can be `OrganizerTableLabels` and `OrganizerTableRow` and such, it would match React's design and make it easier to interact with those as individual components later on, especially as we implement the other preview pages.

Reply:
Agree, make them separate components instead defined outside of the main component